File: automation_web_app/automation-app/src/app/data/Frame.ts
import { BoundingBox } from './BoundingBox';

export class Frame
{
	public boundingBoxes: Array<BoundingBox>
	public data			: string;
	public timeStamp	: string;
	public id			: number;

	constructor()
	{
		this.boundingBoxes = new Array<BoundingBox>();
	}
}

File: automation_web_app/GrayAutomationServer/GrayAutomationServer/Models/Frame.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace GrayAutomationServer.Models
{

public class Frame
{
	public long						Id				{ get; set; }
	public DateTime					TimeStamp		{ get; set; }
	public string					Data			{ get; set; }
	public ICollection<BoundingBox>	BoundingBoxes	{ get; set; }
}

}
